mongo/datadog_checks/mongo/api.py

```diff
@@ -229,6 +229,9 @@ def is_collection_sharded(self, db_name, coll_name):
             self._log.warning("Could not determine if collection %s.%s is sharded: %s", db_name, coll_name, e)
             return False
 
+    def explain_command(self, db_name, command, verbosity="executionStats", session=None):
```

Comment:
Shall we add typing to the function's arguments and return value?

Reply:
i will do it in a separate pr because the entire api.py does not have type hint